doc/editor-integration.md

```diff
@@ -359,3 +365,4 @@ and error counts on the message line:
 ```kak
 define-command -hidden -override lint-show-counters %{}
 ```
+::q!::q
```

Comment:
@brcosta What is this? Have you been editing in vim? ;) Can you undo the changes that are not about clojure extras?

Reply:
haha sorry man, will do